### PR TITLE
PMM-9371 doc: update home-dashboard image

### PR DESCRIPTION
**Pull request by rasika-chivate**
Update the Home Dashboard image on the Welcome page.

https://jira.percona.com/browse/PMM-9371

### CODE REVIEW

**Review by rnovikovP — APPROVED** [review]

**Comment by rasika-chivate** [comment]
The link to preview the changes:

https://new.percona.com/doc/percona-monitoring-and-management/2.x/index.html

File: docs/index.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ PMM helps you improve the performance of your databases, simplify their manageme
 Here's how the home page looks on our <a href='https://pmmdemo.percona.com/' target='_blank'>free, live demo system</a>.
 
 <a href='https://pmmdemo.percona.com/' target='_blank'>
-    <img src="_images/PMM_Home_Dashboard.jpg" alt="PMM Demo Home Page" width=600px class="imgcenter"/>
+    <img src="_images/PMM_Home_Dashboard_1.png" alt="PMM Demo Home Page" width=800px class="imgcenter"/>
```
Comment (denisok) [inline docs/index.md:19]: I see you have updated image name here, but also updated the original image and added "updated" images. Why those changes are needed ?

Comment (rasika-chivate) [reply docs/index.md:19]: I was not getting the desired image quality. I was trying with a different resolution. So during the trial and error method, I did that.

Comment (denisok) [reply docs/index.md:19]: ok, so maybe you need to replace original image and don't add other one's ?

**Review by denisok — APPROVED** [review]

**Pull request merged**